src/compiler-documenting.md

```diff
@@ -35,28 +35,18 @@ Much like individual tests or building certain components you can build only
 
 ## Document internal rustc items
 
-Compiler documentation is not built by default. There's a flag in
-config.toml for achieving the same.
-But, when enabled, compiler documentation does include internal items.
+Compiler documentation is not built by default. To enable it:
 
-Next open up config.toml and make sure these two lines are set to true:
-
-```bash
-docs = true
+```toml
+[build]
 compiler-docs = true
```

Comment:
It is useful to know that this flag is in `config.toml`.

Reply:
oh, indeed